src/Bridges/Constraint/bridges/table.jl

```diff
@@ -0,0 +1,173 @@
+    z::Vector{MOI.VariableIndex}
+    unity::MOI.ConstraintIndex{MOI.ScalarAffineFunction{T},MOI.EqualTo{T}}
+    assignment::Vector{
+        MOI.ConstraintIndex{MOI.ScalarAffineFunction{T},MOI.EqualTo{T}},
+    }
+    # Cache to simplify MOI.get
+    f::F
+    s::MOI.Table{T}
+end
+
+const TableToMILP{T,OT<:MOI.ModelLike} =
+    SingleBridgeOptimizer{TableToMILPBridge{T},OT}
+
+function bridge_constraint(
+    ::Type{TableToMILPBridge{T,F}},
+    model::MOI.ModelLike,
+    f::F,
+    s::MOI.Table{T},
+) where {T,F}
+    m = size(s.table, 1)
+    z = MOI.add_variables(model, m)
+    MOI.add_constraint.(model, z, MOI.ZeroOne())
+    # ∑z_j = 1
+    unity = MOI.add_constraint(
+        model,
+        MOI.ScalarAffineFunction(MOI.ScalarAffineTerm.(one(T), z), zero(T)),
+        MOI.EqualTo(one(T)),
+    )
+    # ∑t_ij z_j == f_i
+    assignment =
+        MOI.ConstraintIndex{MOI.ScalarAffineFunction{T},MOI.EqualTo{T}}[]
+    for (i, fi) in enumerate(MOI.Utilities.eachscalar(f))
+        terms = MOI.ScalarAffineTerm{T}[]
+        for j in 1:m
+            if !iszero(s.table[j, i])
+                push!(terms, MOI.ScalarAffineTerm(s.table[j, i], z[j]))
+            end
+        end
+        assign_f = MOI.ScalarAffineFunction(terms, zero(T))
+        assign_f = MOI.Utilities.operate!(-, T, assign_f, fi)
+        ci = MOI.add_constraint(model, assign_f, MOI.EqualTo(zero(T)))
+        push!(assignment, ci)
+    end
+    return TableToMILPBridge{T,F}(z, unity, assignment, f, s)
+end
+
+function MOI.supports_constraint(
+    ::Type{<:TableToMILPBridge{T}},
+    ::Type{F},
+    ::Type{MOI.Table{T}},
+) where {T,F<:Union{MOI.VectorOfVariables,MOI.VectorAffineFunction{T}}}
+    return true
+end
+
+function MOI.Bridges.added_constrained_variable_types(
+    ::Type{<:TableToMILPBridge},
+)
+    return Tuple{Type}[]
+end
+
+function MOI.Bridges.added_constraint_types(
+    ::Type{TableToMILPBridge{T,F}},
+) where {T,F}
+    return Tuple{Type,Type}[
+        (MOI.VariableIndex, MOI.ZeroOne),
+        (MOI.ScalarAffineFunction{T}, MOI.EqualTo{T}),
+    ]
+end
+
+function concrete_bridge_type(
+    ::Type{<:TableToMILPBridge{T}},
+    ::Type{F},
+    ::Type{MOI.Table{T}},
+) where {T,F<:MOI.AbstractVectorFunction}
+    return TableToMILPBridge{T,F}
+end
+
+function MOI.get(
+    ::MOI.ModelLike,
+    ::MOI.ConstraintFunction,
+    bridge::TableToMILPBridge,
+)
+    return bridge.f
+end
+
+function MOI.get(
+    ::MOI.ModelLike,
+    ::MOI.ConstraintSet,
+    bridge::TableToMILPBridge,
+)
+    return bridge.s
+end
+
+function MOI.delete(model::MOI.ModelLike, bridge::TableToMILPBridge)
+    MOI.delete(model, bridge.assignment)
+    MOI.delete(model, bridge.unity)
+    MOI.delete(model, bridge.z)
+    return
+end
+
+function MOI.get(bridge::TableToMILPBridge, ::MOI.NumberOfVariables)::Int64
+    return length(bridge.z)
+end
+
+function MOI.get(bridge::TableToMILPBridge, ::MOI.ListOfVariableIndices)
+    return copy(bridge.z)
```

Comment:
Don't think we need to copy, we don't copy in other bridges nor in MOI models

Reply:
All the other bridges `copy` or create new vectors.